mobile/android/app/src/main/AndroidManifest.xml

```diff
@@ -31,7 +31,9 @@
                 <action android:name="android.intent.action.VIEW" />
                 <category android:name="android.intent.category.DEFAULT" />
                 <category android:name="android.intent.category.BROWSABLE" />
-                <data android:scheme="aeris" android:host="aeris.com" />
+                <data android:scheme="http" android:host="aeris.com" />
```

Comment:
You should not bind an host that you don't own. Aeris.com is a real website, do not steal it.